CHANGELOG.md

```diff
@@ -3,6 +3,47 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.1
+
+_Dec 17, 2021_
+
+A big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🖨️ Improve the print export to break the pages correctly
+- 🎁 Add `pinnable` option to `GridColDef`
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.2.1` / `@mui/x-data-grid-pro@v5.2.1`
+
+#### Changes
+
+- [DataGridPro] Add `pinnable` option (#3425) @m4theushw
+- [DataGridPro] Avoid filtering columns if no column is pinned (#3438) @m4theushw
+- [DataGrid] Avoid page break inside a row in the Print Export (#3302) @flaviendelangle
+- [DataGrid] Fix `GridEditDateCell` to handle `editRowsModel` correctly (#3267) @alexfauquette
+- [DataGrid] Refactor keyboard/click event management (#3275) @alexfauquette
+- [DataGrid] Fire change event when the state changes, instead of when the prop changes (#3388) @flaviendelangle
+- [DataGrid] Unsubscribe event listeners registered in uncommitted renders (#3310) @m4theushw
+- [DataGrid] Improve German (de_DE) locale (#3430) @sebastianfrey
+- [DataGrid] Improve Hebrew (he-IL) locale (#3445) @ColdAtNight
+- [DataGrid] Improve Dutch (nl-NL) locale (#3429) @jaapjr
```

Comment:
```suggestion
- [l10n] Improve German (deDE) locale (#3430) @sebastianfrey
- [l10n] Improve Hebrew (heIL) locale (#3445) @ColdAtNight
- [l10n] Improve Dutch (nlNL) locale (#3429) @jaapjr
```

Reply:
Are you sure? In previous changelog, translation was with [DataGrid] tag

Reply:
I changed their tags to be "[DataGrid]" in the CHANGELOG, but the PR is "[l10n]", which is the same approach taken by the core.